source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -51,6 +51,13 @@ class UIACustomAttributeID(enum.IntEnum):
 	COLUMN_NUMBER = 2
 	SECTION_NUMBER = 3
 	BOOKMARK_NAME = 4
+	COLUMNS_IN_SECTION = 5
```

Comment:
Is this supposed to be included in this PR?

Reply:
It is not directly required, however it is definitely part of the official enum that Microsoft has provided, and we will most likely use that value in the near future. So I figure it makes sense to include it for completeness.